snippets/snippets.json

```diff
@@ -52,5 +52,50 @@
             "}"
         ],
         "description": "A hasOneThrough relationship function"
+    },
+    "A MorphOne relationship function": {
+        "prefix": "morphOne",
+        "body": [
+            "public function ${1:name}()\n{",
+            "    return \\$this->morphOne(${2:class}, '${1:name}able');",
+            "}"
+        ],
+        "description": "A morphOne relationship function"
+    },
+    "A MorphTo relationship function": {
+        "prefix": "morphTo",
+        "body": [
+            "public function ${1:nameable}()\n{",
+            "    return \\$this->morphTo(${2});",
+            "}"
+        ],
+        "description": "A morphTo relationship function"
+    },
+    "A MorphMany relationship function": {
+        "prefix": "morphMany",
+        "body": [
+            "public function ${1:names}()\n{",
+            "    return \\$this->morphMany(${2:class}, '${3:nameable}');",
```

Comment:
This "namable" parameter doesn't work as expected, it does not get the same value as the function name when filling in the function name, it should be '${1:name}able. make sure to fix this for the other polymorphic relationships.

Reply:
```php
public function comments()
    {
        return $this->morphMany(Comment::class, 'commentable');
    }
```

It might work for comments easily, but might have problem with something like `tags` Hence I preferred to keep it separate.